scripts/appbuilder

```diff
@@ -8,12 +8,10 @@
 function print_usage {
     echo "$(basename $0): a useful shell script for converting Docker images into Mystikos cpio archives"
     echo
-    echo "Usage: $(basename $0) [-o file] [-d [file]] [[-i] image] [-e extras] [-v]"
+    echo "Usage: $(basename $0) [-o file] [-i image] [-e extras] [-v] [[-d] file]"
     echo
     echo "  -d file     Build docker image from the named file"
-    echo "              Defaults to './Dockerfile' if no option specified"
     echo "  -i image    Specify an existing Docker image name to build from"
-    echo "              If neither -d nor -i are specified, will look for Dockerfile in current dir"
     echo "  -f format   Specify the output format. Options: dir, cpio. Default: 'dir'"
     echo "  -e extras   Specify the extra options while building the container image"
     echo "  -o      Specify the name of the output file or directory"
```

Comment:
Please provide a few example usages here.

Reply:
That depends on the definition for the appbuilder args. After the definition(sequence, default option) has been confirmed, I will add examples.